src/bors/mod.rs

```diff
@@ -128,25 +127,19 @@ pub fn create_merge_commit_message(pr: handlers::PullRequestData, merge_type: Me
         MergeType::Auto => pr.db.approver().unwrap_or("<unknown>"),
     };
 
-    let mut pr_description = suppress_github_mentions(&pr.github.message);
-    match &merge_type {
-        // Strip all PR text for try builds, to avoid useless issue pings on the repository.
+    let pr_description = match &merge_type {
         // Only keep any lines starting with `CUSTOM_TRY_JOB_PREFIX`.
-        MergeType::Try { try_jobs } => {
-            // If we do not have any custom try jobs, keep the ones that might be in the PR
-            // description.
-            pr_description = if try_jobs.is_empty() {
-                pr_description
-                    .lines()
-                    .map(|l| l.trim())
-                    .filter(|l| l.starts_with(CUSTOM_TRY_JOB_PREFIX))
-                    .join("\n")
-            } else {
-                // If we do have custom jobs, ignore the original description completely
-                String::new()
-            };
-        }
-        MergeType::Auto => {}
+        MergeType::Try { try_jobs } if try_jobs.is_empty() => pr
+            .github
+            .message
+            .lines()
+            .map(|l| l.trim())
+            .filter(|l| l.starts_with(CUSTOM_TRY_JOB_PREFIX))
+            .join("\n"),
+        // If we do not have any custom try jobs, keep the ones that might be in the PR
```

Comment:
The comments got a bit juggled around here :) This comment should be above the first match arm, and the "If we do have custom jobs..." comment is missing (it should be readded here).